pkgs/games/keeperrl/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, SDL2, SDL2_image, openal, libvorbis, zlib, curl,
+  libGL, makeWrapper, boost }:
+
+stdenv.mkDerivation rec {
+  name = "keeperrl-${version}";
+  version = "alpha24";
+
+  src = fetchFromGitHub {
+    owner = "miki151";
+    repo = "keeperrl";
+    rev = "${version}";
+    sha256 = "00vpq4njdl8hqndjvx2r0w0azz9lfil4qg1rlhj3y9qvc1i0xwwn";
+  };
+
+  patches = [ ./makefile_cflags.patch ];
+
+  buildInputs = [
+    SDL2 SDL2_image openal libvorbis zlib curl libGL makeWrapper
+  ] ++ stdenv.lib.optional stdenv.isDarwin boost ;
+
+  makeFlags = [ "OPT=true" "RELEASE=true" ];
+
+  CFLAGS = "-I${SDL2.dev}/include/SDL2";
+
+  installPhase = ''
+    mkdir -p $out/keeperrl
```

Comment:
Should store data files in `$out/share/keeperrl` instead. And you can use a single line:
```
mkdir -p $out/share/keeperrl $out/bin
```

Reply:
Done. Thanks.